gobblin-temporal/src/main/java/org/apache/gobblin/temporal/ddm/activity/impl/ProcessWorkUnitImpl.java

```diff
@@ -62,11 +61,14 @@
 public class ProcessWorkUnitImpl implements ProcessWorkUnit {
   private static final int LOG_EXTENDED_PROPS_EVERY_WORK_UNITS_STRIDE = 100;
 
+  private static final String MAX_SOURCE_PATHS_TO_LOG_PER_MULTI_WORK_UNIT = ProcessWorkUnitImpl.class.getName() + ".maxSourcePathsToLogPerMultiWorkUnit";
+  private static final int DEFAULT_MAX_SOURCE_PATHS_TO_LOG_PER_MULTI_WORK_UNIT = 5;
+
   @Override
   public int processWorkUnit(WorkUnitClaimCheck wu) {
     try (FileSystem fs = Help.loadFileSystemForce(wu)) {
       List<WorkUnit> workUnits = loadFlattenedWorkUnits(wu, fs);
-      log.info("WU [{}] - loaded {} workUnits", wu.getCorrelator(), workUnits.size());
+      log.info("(M)WU [{}] - loaded; found {} workUnits", wu.getCorrelator(), workUnits.size());
```

Comment:
Is it guaranteed to be multiworkunits here? I think that depends if the job has enough workunits to pack?

Reply:
exactly, hence I put "(M)WU" to cover the case equally of "MWU" or "WU".  would you prefer I write it as:
```
log.info("WU/MWU [{}] - loaded; found {} workUnits",  ...);
```
?

Reply:
Ah I think its fine the way it is then, just misunderstood what the brackets meant originally